safetensors/benches/benchmark.rs

```diff
@@ -50,6 +61,30 @@ pub fn bench_deserialize(c: &mut Criterion) {
     });
 }
 
+pub fn bench_deserialize_shallow(c: &mut Criterion) {
```

Comment:
There is no problem when deserializing raw tensors, so added this to verify and keep it regression free.